reviewdog.go

```diff
@@ -15,16 +15,21 @@ import (
 // or linter, get diff and filter the results by diff, and report filterd
 // results.
 type Reviewdog struct {
-	p Parser
-	c CommentService
-	d DiffService
+	toolname string
+	p        Parser
+	c        CommentService
+	d        DiffService
 }
 
+var ExportedVarWithoutComment = 1 // TODO: revert
+
 // NewReviewdog returns a new Reviewdog.
-func NewReviewdog(p Parser, c CommentService, d DiffService) *Reviewdog {
+func NewReviewdog(toolname string, p Parser, c CommentService, d DiffService) *Reviewdog {
 	return &Reviewdog{p: p, c: c, d: d}
 }
 
+var unusedVar = 2 // TODO: revert
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
var unusedVar is unused